InputHandler.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+/**
+ * Defines an entry point.
+ * @author Yury Suponev
+ */
+public class InputHandler {
+
+  private static final int SIDES_NUMBER = 3;
+  private static final String INVALID_INPUT = "Invalid inputs. Try again using only positive numbers.";
+
+  /**
+   * Reads input numbers.
+   * @param scanner Scanner object that is used to read numbers from console
+   * @param sidesLength Length of each triangle side
+   */
+  private static void scan(Scanner scanner, Double[] sidesLength) {
+    int i = 0;
+    while (i < SIDES_NUMBER) {
+      try {
+        sidesLength[i] = Double.parseDouble(scanner.next());
+        if (!sidesLength[i].isInfinite() && sidesLength[i] > 0) {
+          i++;
+        } else {
+          System.out.println(INVALID_INPUT);
+          i = 0;
+        }
+      } catch (NumberFormatException e) {
+        System.out.println(INVALID_INPUT);
+        i = 0;

```

Comment:
Does it mean that application force to re-enter all 3 parameters if user entered 2 correct and the last wrong? Probably there should be some message that shows which parameter user is entering